### ISSUE TITLE
[1.16.4] Durability loss

### ISSUE THREAD

**Issue by Ruchian** [zlainsama/CosmeticArmorReworked#55]
Minecraft: 1.16.4
Forge 35.1.13
Cosmetic armor: 1.16.4-v1

Situation: Dedicated server
Modpack: All The Mods 6, 1.3.7

Issue: Equipment placed in the cosmetic armor slots lose durability when you take damage.
Expected: No such thing to happen.

Reason for expectation: All of the enchants on these armor pieces are ignored. Meaning its like you're not wearing them. This includes Unbreaking which can have an overpowered level due to a mod called Apotheosis. Meaning, wearing it directly yourself is more efficient durability wise.

I don't know if durability loss is a feature or a bug, so I am reporting it anyway. If it is a feature, I don't get why it ignores enchants. My poor netherrite armor.

**Comment by zlainsama** [comment]
If true, it must be a bug. They should not be affected nor affecting the game logic.
Could you try to make this happen on other armor reliably? if it was only an one time thing, it might just be network desync.

**Comment by Ruchian** [comment]
I can't reproduce it.
I tried all kinds of things.
The server I was playing on was having issues (it also crashed a number of times), what you said could be the case and perhaps it was just visual. I currently do not lag as much on the same server, so I can't reliably reproduce it nor find a cause.
My armor currently remains at 135 durability no matter what I try at least.
Hm... sorry, perhaps I over reacted by reporting it.

I was having a hard time due to other bugs and issues (including that crash problem), the frustration made me feel like reporting everything I experienced that felt off basically, I suppose I should have tested it in singleplayer before reporting it or something. Aaa- sorry.
